src/services/apps-metadata.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+export function fetchEnv() {
+    return fetch('env.json').then((res) => res.json());
+}
```

Comment:
ping @jonenst @flomillot

what do you think?

Reply:
IMO the function should stay here, we can eventually put the location in the common redux store

Reply:
huh... I don't think that is what we want to do here: if i don't mistake, `env.json` is meant to be possibly dynamic if we want without having to reload/rebuild the app/container.

Reply:
I see with Jon, and `env.json` like others files in `public` folders are meant to be accessible from "http://host/env.json", while the content must be fetched each time because the content can be modified by the deployment.

Reply:
Being dynamic does not mean the location change. And it's the same problem being here or in gridstudy no ?
Or I missed something ? I admit I don't understand the problem in this case.